.gitignore

```diff
@@ -0,0 +1,5 @@
+/build*
+.vscode
+__pycache__
+
+/platforms
```

Comment:
I think platforms are a submodule, so shouldn't be a problem?

Reply:
The vck190_bare platform is not a submodule, which generates build files of petalinux and vivado after making.

Reply:
It generates them in the platforms directory? That seems.. wrong...

Reply:
Right. Based on the [document](https://xilinx.github.io/mlir-aie/Platform.html), we do `make all` under `platforms/vck190_bare` to built the petalinux and vivado project. The generated files are all under `platforms/vck190_bare`.

Reply:
Can you make the ignore pattern more selective then?  Ultimately, this seems like something that should be fixed, though. so we're not generating things in the source tree.